packages/components/src/radio-group/stories/index.story.tsx

```diff
@@ -58,7 +66,9 @@ export const Disabled = () => {
 };
 
 const ControlledRadioGroupWithState = () => {
-	const [ checked, setChecked ] = useState( 1 );
+	const [ checked, setChecked ] = useState<
+		RadioProps[ 'value' ] | undefined
```

Comment:
Nit: I usually prefer to avoid importing prop types directly in Storybook when possible. We could alternatively do something like

```suggestion
		React.ComponentProps< typeof Radio >[ 'value' ],
```